packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -222,47 +343,48 @@ class _TreeTableState<T extends TreeNode<T>> extends State<TreeTable<T>>
   }
 }
 
-/// Presents a [TreeNode].
+/// Presents a [node] as a row in a table.
 ///
 /// When the given [node] is null, this widget will instead present
 /// column headings.
-class _TreeNodeWidget<T extends TreeNode<T>> extends StatefulWidget {
-  /// Constructs a [_TreeNodeWidget] that presents the column values for
+class _TableRow<T> extends StatefulWidget {
+  /// Constructs a [_TableRow] that presents the column values for
   /// [node].
-  const _TreeNodeWidget({
+  const _TableRow({
     @required Key key,
     @required this.node,
     @required this.tableState,
   }) : super(key: key);
 
-  /// Constructs a [_TreeNodeWidget] that presents the column titles instead
+  /// Constructs a [_TableRow] that presents the column titles instead
   /// of any [TreeNode].
```

Comment:
feels like this comment shouldn't reference TreeNode if it is in a general table row class

Reply:
Cleaned up as part of rewriting this.